tutorials/preprocessing/working_with_sensor_locations.py

```diff
@@ -0,0 +1,145 @@
+# as whether to display channel names or just points (the ``show_names``
+# parameter) and whether to display sensor positions in 3D or as a 2D topomap
+# (the ``kind`` parameter):
+
+ten_twenty_montage = mne.channels.read_montage('standard_1020')
+ten_twenty_montage.plot(show_names=False)
+fig = ten_twenty_montage.plot(kind='3d')
+fig.gca().view_init(azim=70, elev=15)
+
+###############################################################################
+# Similar functionality is also available with the
+# :meth:`~mne.io.Raw.plot_sensors` method of :class:`~mne.io.Raw` objects,
+# again with the option to plot in either 2D or 3D.
+# :meth:`~mne.io.Raw.plot_sensors` also allows channel selection by type, can
+# color-code channels in various ways (by default, channels listed in
+# ``raw.info['bads']`` will be plotted in red), and allows drawing into an
+# existing matplotlib ``axes`` object (so the channel positions can easily be
+# made as a subplot in a multi-panel figure):
+
+fig = plt.figure()
+ax2d = fig.add_subplot(121)
+ax3d = fig.add_subplot(122, projection='3d', aspect='equal')
+raw.plot_sensors(ch_type='eeg', axes=ax2d)
+raw.plot_sensors(ch_type='eeg', axes=ax3d, kind='3d')
+ax3d.view_init(azim=70, elev=15)
+
+###############################################################################
+# Reading sensor digitization files
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# It's probably evident from the 2D topomap above that there is some
+# irregularity in the sensor positions — this is because the sensor positions
+# stored in the :class:`~mne.io.Raw` object are based on digitizations of the
+# sensor positions on an actual subject's head. Sensor digitizations are read
+# with :func:`mne.channels.read_dig_montage` and added to :class:`~mne.io.Raw`
+# objects with the :meth:`~mne.io.Raw.set_montage` method. See the function
+# documentation for further details.
+#
+# .. TODO sample data doesn't have separate .hsp or .elp files, so can't demo
+#    this functionality
+#
+#
+# Rendering sensor position with mayavi
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# It is also possible to render an image of the sensor helmet in 3D, using
+# mayavi instead of matplotlib, by calling the :func:`mne.viz.plot_alignment`
+# function:
+
+fig = mne.viz.plot_alignment(raw.info, trans=None, dig=False, eeg=False,
+                             surfaces=[], meg=['helmet', 'sensors'],
+                             coord_frame='meg')
+mlab.view(azimuth=50, elevation=90, distance=0.5)
+
+###############################################################################
+# :func:`~mne.viz.plot_alignment` requires an :class:`~mne.Info`
+# object, and can also render MRI surfaces of the scalp, skull, and brain,
+# making it useful for :ref:`assessing coordinate frame transformations
+# <plot_source_alignment>`.
+#
+#
+# TODO
+# ^^^^
+#
+# - mne.channels.make_eeg_layout() ?
+# - mne.channels.make_1020_channel_selections() ?
```

Comment:
This function is very specific to a subset of EEG analyses, and not at the stage of preparing MNE instances, I would say. (At least that's how I intended it :)) Probably not optimal here.

Reply:
OK, will skip `make_1020_channel_selections` here.